### DISCUSSION TITLE
esp8266 flashing help

### DISCUSSION THREAD

**Discussion by jnthnplbkr** [universam1/iSpindel#534] [Q&A]
I am trying to flash the ESP8266 with no luck.

I'm following instructions and don't really know how this is done so I have no idea if I'm doin git right.

I have a kit assembled, battery out, flashing tool open, firmware bin file loaded (7.1.2)

Status doesn't change and I get these two messages.

1. require(("wifi")
2. connect.world()

any advice or direction to it would be appreciated.

thanks
jnthnplbkr

**Comment by wilbrod** [comment]
Do you see the com port being added to the drop down list when you connect
your WEMOS board to your PC? If not you probably need to install the driver
or you need a data capable Micro USB cable.

**Reply by jnthnplbkr** [reply]
com port is always 3...doesn't change like I understand it should. I'll try another cable.

thanks for the quick reply

**Reply by jnthnplbkr** [reply]
I'm using a data cable...480 Mbps

**Comment by thegreatgunbantoad** [comment]
This tool is quite good:
https://github.com/thorrak/brewflasher/releases/tag/v1.0.1
You can select for it to just use the first port it can find a device on.

**Reply by jnthnplbkr** [reply]
appreciate the help. I tried the brewflasher tool...got this...starting to think I have some assembly issues?

_Downloading firmware...
Downloaded successfully!

Command: esptool.py --chip esp8266 write_flash --flash_mode dio 0x00000 C:\Users\Gerd\AppData\Local\Temp\firmware.bin --erase-all

esptool.py v2.8
Found 0 serial ports_

**Reply by thegreatgunbantoad** [reply]
If you can whip the ESP board off,  I believe it should be flashable on it's own.  They are a sod to get off though.  I'll see if I have a spare one knocking about to test.

**Comment by wilbrod** [comment]
When it is connected to your computer, is the WEMOS d1 mini detected in the
devices ? Or is it unknown? If unknown you need to install the driver.
